lib/entity.js

```diff
@@ -29,29 +51,66 @@ function Entity (entity) {
   return this;
 }
 
-Object.defineProperty(Entity.prototype, 'level', {
-  get: function level () {
-    if (this.experience) {
-      return Math.floor(Math.log(this.experience / 10));
-    } else {
-      return 0;
-    }
+Object.defineProperty(Entity.prototype, 'attack', {
+  get: function attack () {
+    Entity.derrivedStats.damage = Entity.stats.attack;
+
+    return Entity.stats.attack;
+  }
+});
```

Comment:
You don't need to use `Object.defineProperty()` if the value isn't calculated on the fly.  Just do something like this:
```suggestion
this.attack = `foo`;
```

If you're just looking for the raw statistic, you can remove these entirely.